### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by jiakeliu** [gunnertech/thumbwar-ios-client#130]
User needs to get notifications for:

1. When an opponent challenges him
2. When an opponent declared a winner
3. When a war closes (winner declared)

**Comment by CodySwannGT** [comment]
This is the case with the API - however, We almost definitely need new APNS certs.

I think that's all you have to do - generate the new cert and then I'll install it on the heroku server or show you how to do it.

**Comment by CodySwannGT** [comment]
This is a perfect tutorial: http://www.raywenderlich.com/32960/apple-push-notification-services-in-ios-6-tutorial-part-1

**Comment by husseinm** [comment]
Yes I need to update the APNS certs, we are not talking about push notifications though. The issue is when I make the API request for notifications, I do not see notifications in the JSON for types # 1 and 2

**Comment by CodySwannGT** [comment]
Got it. That wasn't clear from the description. I thought we were calling those "Alerts"?

**Comment by husseinm** [comment]
Yeah, Sorry about that. You are right they are called Alerts, at least that is what the view is called...

**Comment by CodySwannGT** [comment]
No worries. Just want to make sure we're all on the same verbiage

**Comment by CodySwannGT** [comment]
1 and 2 were already there. 3 is deploying now

**Comment by husseinm** [comment]
I think this was fixed by Cody

**Comment by jiakeliu** [comment]
This is about the events showing up on the Alerts page, right? Do we have a ticket to fix the push notifications?

**Comment by husseinm** [comment]
Yes and No, we will use this for both issues.

**Comment by jiakeliu** [comment]
okay, so i'll assign the ticket back to you until the push notification is fixed.
